arch-template.json

```diff
@@ -78,6 +78,9 @@
       }, {
         "type": "vagrant-cloud",
         "box_tag": "SimonPe/archlinux",
+        "only": [
+          "archlinux-kvm"
+        ],
```

Comment:
As I will be making lot's of changes to the archlinux-virtualbox it isn't quite ready to be published to vagrant-cloud yet.